core/commands/mv.py

```diff
@@ -0,0 +1,35 @@
+import functools
+import os
+import sublime
+from sublime_plugin import WindowCommand
+
+from ..git_command import GitCommand
+
+
+class GsMvCurrentFileCommand(WindowCommand, GitCommand):
+
+    """
+    Prompt the user for a new name for the current file.
+    """
+
+    def run(self):
+        if self.file_path:
+            parent, base_name = os.path.split(self.file_path)
+            on_done = functools.partial(self.on_done,
+                self.file_path, parent, base_name)
+            v = self.window.show_input_panel("New Name:", base_name,
+                on_done, None, None)
+            name, ext = os.path.splitext(base_name)
+            v.sel().clear()
+            v.sel().add(sublime.Region(0, len(name)))
```

Comment:
I meant it here, but as you stated. You only select the name without the extension.